### PR TITLE
March demo summary

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

**Review by saulaski — APPROVED** [review]

**Review by matks — APPROVED** [review]

File: news/_posts/public-oss-demo/2022-03-02-public-demo-2-2022.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "PrestaShop Project Public Demo - March 2022 session"
+subtitle: "We held the second session of this year open source project's public demo"
+date:   2022-03-03 15:00
```
Comment (matks) [inline news/_posts/public-oss-demo/2022-03-02-public-demo-2-2022.md]:
```suggestion
date:   2022-03-03 17:30
```

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @kpodemski

**Comment by matks** [comment]
I do not see the post on https://build.prestashop.com/

Something wrong happened

**Comment by matks** [comment]
https://github.com/PrestaShop/prestashop.github.io/issues/1049

**Pull request merged**